+sig/+node/Signal.m

```diff
@@ -308,31 +308,35 @@
     end
     
     function tr = applyTransferFun(varargin)
-      % New signal derived by applying a transfer function to input nodes
+      % New signal derived by applying a transfer function to input node(s)
+      % 
+      % This function creates a node from input values (which can be
+      % signals or non-signals), then creates a new signal containing the
+      % new node.
+      % 
+      % Inputs:
+      %   `varargin`: contains one (or more) input values/signals, `sigs`, 
+      %   used to create the output signal; a string, `funName`, of the
+      %   transfer function; an optional function handle, `funArg`, which
+      %   can be applied by the transfer function, and an optional string
+      %   `formatSpec`, which is used to format the name of the output
+      %   signal
       %
-      % [tr] = s1.applyTransfer([s2], ..., funName, funArg, formatSpec)
```

Comment:
Looks like we've lost information that was previously in the doc string.  Can you leave this in?  
Original doc string: https://github.com/dendritic/signals/blob/c2cf2854491525905ed9f21e0d708b451bfce988/%2Bsig/%2Bnode/Signal.m#L307-L324

Reply:
I removed this because I give concrete examples, as per one of your earlier suggestions : )

Reply:
I'll put this line back in as the first line in the docstring

Reply:
What was your reasoning for removing this information:

```
      % [...] Transfer functions work at a lower
      % level than transformations like like map or mapn, instead operating
      % directly with the underlying input nodes and output node,
      % potentially using both their current *and* new values.
```
I think the original definition of "returns a new signal whose values result from applying a transfer function to a variable number of input signals or constant values" is a little clearer.  "This function creates a node from input values (which can be signals or non-signals)" is a little ambiguous (new nodes aren't created from the input signals).